llm-server/src/model/remote-model-factory.ts

```
@@ -0,0 +1,34 @@
+    modelName: string,
+  ): Promise<ModelInstance> {
+    if (!config.endpoint || !config.token) {
+      throw new Error('Remote model requires endpoint and token configuration');
+    }
+
+    const key = `${config.endpoint}:${config.token}:${modelName}`;
+    if (!this.instances.has(key)) {
+      const instance = new RemoteOpenAIModelEngine(config);
+      this.instances.set(key, instance);
+    }
+    return this.instances.get(key)!;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)